kits/daisy/CMakeLists.txt

```diff
@@ -0,0 +1,136 @@
+cmake_minimum_required (VERSION 3.1)
+project (hexapod)
+
+SET (CMAKE_CXX_STANDARD 11)
+SET (CMAKE_CXX_STANDARD_REQUIRED ON)
+SET (CMAKE_CXX_FLAGS "-O3 -g")
+SET (SONAME "hebi")
+
+# Instruct CMake to run moc automatically when needed.
+set(CMAKE_AUTOMOC ON)
+
+# Find Qt libraries
+find_package(Qt5 REQUIRED COMPONENTS Core Gui Widgets NO_CMAKE_PATH)
+# NOTE: to add support for 3D Qt components, replace the above with the
+# following line
+#find_package(Qt5 REQUIRED COMPONENTS Core Gui Widgets 3DCore 3DExtras 3DRender 3DInput NO_CMAKE_PATH)
+
+find_package(Doxygen)
+
+# Get the C++ API directory:
+set(ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../..)
+get_filename_component(ROOT_DIR "${ROOT_DIR}" REALPATH)
+# `HEBI_CPP_DIR` can be used to point to an already downloaded C++ API.
+# This is useful for environments that would prefer not to download files
+# from the internet (or even don't have internet access).
+if(NOT HEBI_CPP_DIR)
+  set(HEBI_DIR ${ROOT_DIR}/hebi-cpp)
+  include(../../projects/cmake/DownloadHebiCpp.cmake)
+else()
+  set(HEBI_DIR ${HEBI_CPP_DIR})
+endif()
+
+set(hebi_cpp_build_dir hebi-cpp-${CMAKE_BUILD_TYPE})
+string(TOLOWER ${hebi_cpp_build_dir} hebi_cpp_build_dir)
+get_filename_component(hebi_cpp_build_dir "${CMAKE_CURRENT_BINARY_DIR}/${hebi_cpp_build_dir}" REALPATH)
+add_subdirectory(${HEBI_DIR} ${hebi_cpp_build_dir})
+
+include_directories (
+  ${CMAKE_CURRENT_SOURCE_DIR}/src          # Our source files
+  ${HEBI_DIR}/src ${HEBI_DIR}/hebi/include ${HEBI_DIR}/Eigen)
+link_directories (
+  ${HEBI_CPP_LINK_DIRECTORIES})
+
+add_library(hexapod_core OBJECT
```

Comment:
We need to stop doing these `OBJECT` libraries and adhere to Windows' strange standard of requiring `__declspec(dllexport)` on exported elements

Reply:
This was mainly as a comment; there's nothing to resolve with this right now